core/validation/__init__.py

```diff
@@ -28,13 +27,11 @@ def tf_validate(
     :return: any captured validation failures
     """
     cast_to_schema(data_dict, validation_schema)
-    validation_failures_list: list[SchemaUserValidationFailure | InternalValidationFailure] = validate_data(
-        data_dict, validation_schema
-    )
+    validation_failures = validate_data(data_dict, validation_schema)
 
     if fund_specific_validation:
         fund_specific_failures = fund_specific_validation(data_dict, original_workbook)
-        validation_failures: list[Any] = [*validation_failures_list, *fund_specific_failures]
```

Comment:
I'd rather not change the type of `validation_failures` and overwrite it mid-function. Can we do something here that keeps `validation_failures_list` existing - or refactors this in some other cleaner way?

Reply:
Changed it back as it was and added an ignore type-error comment.